ee/tables/tablehelpers/exec.go

```diff
@@ -19,66 +20,80 @@ import (
 // An example of this is to run the exec as a specific user instead of root.
 type ExecOps func(*exec.Cmd) error
 
-// Exec is a wrapper over exec.CommandContext. It does a couple of
-// additional things to help with table usage:
-//  1. It enforces a timeout.
-//  2. Second, it accepts an array of possible binaries locations, and if something is not
-//     found, it will go down the list.
-//  3. It moves the stderr into the return error, if needed.
-//
-// This is not suitable for high performance work -- it allocates new buffers each time.
+func WithDir(dir string) ExecOps {
+	return func(cmd *exec.Cmd) error {
+		cmd.Dir = dir
+		return nil
+	}
+}
+
+func WithAppendEnv(key, value string) ExecOps {
+	return func(cmd *exec.Cmd) error {
+		cmd.Env = append(cmd.Env, fmt.Sprintf("%s=%s", key, value))
+		return nil
+	}
+}
+
+// RunSimple is a wrapper over allowedcmd.AllowedCommand.
+// It enforces a timeout, logs, traces, and returns the stdout as a byte slice.
+// It discards stderr.
 //
-// `possibleBins` can be either a list of command names, or a list of paths to commands.
-// Where reasonable, `possibleBins` should be command names only, so that we can perform
-// lookup against PATH.
-func Exec(ctx context.Context, slogger *slog.Logger, timeoutSeconds int, execCmd allowedcmd.AllowedCommand, args []string, includeStderr bool, opts ...ExecOps) ([]byte, error) {
+// This is not suitable for high performance work -- it allocates new buffers each time
+// Use Run() for more control over the stdout and stderr streams.
+func RunSimple(ctx context.Context, slogger *slog.Logger, timeoutSeconds int, cmd allowedcmd.AllowedCommand, args []string, opts ...ExecOps) ([]byte, error) {
```

Comment:
still not 100% sold on the functions .... wonder if we should make an option for `WithStderr(w io.Writer)`

Reply:
TBH I'm not 100% sure either. But I think this is a great step, and we can refactor if it feels wrong.